website/docs/r/autoscale_setting.html.markdown

```diff
@@ -0,0 +1,351 @@
+greater than the delay in metric collection, which can vary from resource-to-resource. Must be between 12 hours 
+and 5 minutes. ISO 8601 duration format.
+
+* `time_aggregation` - (Required) Specifies how the data that is collected should be combined over time. The default 
+value is Average. Possible values are: `Average`, `Minimum`, `Maximum`, `Last`, `Total`, `Count`.
+
+* `operator` - (Required) Specifies the operator that is used to compare the metric data and the threshold. Possible 
+values are: `Equals`, `NotEquals`, `GreaterThan`, `GreaterThanOrEqual`, `LessThan`, `LessThanOrEqual`.
+
+* `threshold` - (Required) Specifies the threshold of the metric that triggers the scale action.
+
+`scale_action` supports the following:
+
+* `direction` - (Required) Specifies the scale direction. Possible values are: `Increase`, `Decrease`.
+
+* `type` - (Required) Specifies the type of action that should occur, this must be set to `ChangeCount` or `PercentChangeCount`.
+
+* `value` - (Required) Specifies the number that is involved in the scaling action. This value must be 1 or greater. The default 
+value is 1.
+
+* `cooldown` - (Required) Specifies the amount of time to wait since the last scaling action before this action occurs. Must 
+be between 1 week and 1 minute. ISO 8601 duration format.
+
+`fixed_date` supports the following:
+
+* `time_zone` - (Required) Specifies the time zone of the start and end times for the profile.
+
+* `start` - (Required) Specifies the start time for the profile.
+
+* `end` - (Required) Specifies the end time for the profile.
+
+`recurrence` supports the following:
+
+* `frequency` - (Required) Specifies how often the schedule profile should take effect. This value must `Week`, meaning each week 
+will have the same set of profile.
+
+* `schedule` - (Required) Specifies the scheduling constraints for when the profile begins.
+
+`schedule` supports the following:
+
+* `time_zone` - (Required) Specifies the time zone for the hours of the profile.
+
+* `days` - (Required) Specifies a collection of days that the profile takes effect on. Possible values are Sunday through Saturday.
+
+* `hours` - (Required) Specifies a collection of hours that the profile takes effect on. Values supported are 0 to 23 on the 24-hour 
+clock (AM/PM times are not supported).
+
+* `minutes` - (Required) Specifies a collection of minutes at which the profile takes effect at.
+
+`notification` supports the following:
+
+* `operation` - (Required) Specifies the type of operation which triggers the notification. Possible value is `Scale`.
+
+* `email` - (Required) Specifies the email notification settings.
+
+* `webhook` - (Optional) Specifies the webhook notification settings.
+
+`email` supports the following:
+
+* `send_to_subscription_administrator` - (Required) Specifies whether to send email notifications to the subscription administrator.
```

Comment:
The default behavior on portal is not sending emails, and since email property is optional, I think it is fine not to set a default value in the schema - if users do need email notifications, they specify email properties.